README.md

```diff
@@ -58,3 +69,16 @@ For macOS:
 cd drake-build
 export PYTHONPATH=${PWD}/install/lib/python3.9/site-packages:$PYTHONPATH
 ```
+
+We have used Mosek to solve most of the examples. To solve using Mosek, you'll need to give Drake access to a Mosek license file as described [here](https://drake.mit.edu/bazel.html#mosek). Mosek provides a personal academic license for free.
+
+You will also need to install `gcs` and its dependencies. You can do this by running
```

Comment:
not `gcs` anymore.  how about
"You will also need to install additional dependencies for this repo"